sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotationVisibility.kt

```diff
@@ -0,0 +1,24 @@
+package com.mapbox.maps.viewannotation
+
+internal enum class ViewAnnotationVisibility {
```

Comment:
Would it make sense to expose this class to user? As its more granular.

Reply:
no, there's no sense at all for user to know such low-level implementation details.
Moreover at least I'm very happy that we will have API we introduced now right now just to know whether view annotation is shown / hidden (bool true|false) when in reality it could be: removed from layout, made invisible, made gone, made visible but not yet positioned etc